Makefile

```diff
@@ -34,7 +35,7 @@ ifeq ($(LEDGER_ENABLED),true)
   endif
 endif
 
-ifeq ($(WITH_CLEVELDB),yes)
+ifeq (cleveldb,$(findstring cleveldb,$(GAIA_BUILD_OPTIONS)))
```

Comment:
Hmmm.. is the use of `GAIA_BUILD_OPTIONS` documented?

Reply:
I have not found any doc but they were removed from the build meanwhile. Dropping them as well.